packages/KTable/KTable.vue

```diff
@@ -131,14 +131,14 @@
             :tabindex="isClickable ? 0 : null"
             :role="isClickable ? 'link' : null"
             v-bind="rowAttrs(row)"
-            v-on="trlisteners(row, 'row')"
+            v-on="hasSideBorder ? tdlisteners(row, row) : null"
```

Comment:
question: why would the click events only be bound if hasSideBorder is true? 

I’m probably missing something, and I know you’re going to review Dustin’s changes, but I want to understand this lol

Reply:
You had mentioned that if we didn't have `border-collapse` (which `hasSideBorder` doesn't) might cause space between cells. Seems like a non-issue though, so backed out the code.